lib/web_ui/test/canvaskit/vertices_test.dart

```diff
@@ -2,17 +2,26 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:typed_data';
+
 import 'package:test/bootstrap/browser.dart';
 import 'package:test/test.dart';
 import 'package:ui/src/engine.dart';
 import 'package:ui/ui.dart' as ui;
+import 'package:web_engine_tester/golden_tester.dart';
 
 import 'common.dart';
 
 void main() {
   internalBootstrapBrowserTest(() => testMain);
 }
 
+final CkVertices circleVertices = ui.Vertices.raw(
```

Comment:
This may be problematic as `setUpCanvasKitTest` deletes any Skia objects between tests to avoid cross-test pollution. Would it be possible to make it a function that returns a new instance every time?

Reply:
I tried that and it still failed